src/index.html

```diff
@@ -5,8 +5,429 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="font">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <div class="nav_container">
+        <div class="logo-img-container">
+          <img
+            class="logo_img"
+            src="images/logo.png"
+            alt="logo"
+          >
+        </div>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a class="nav__link is-active"href="http://localhost:8080/">apple</a>
```

Comment:
```suggestion
              <a class="nav__link is-active" href="#">apple</a>
```
No need to hardcore the URL for localhost, just use hashtag instead